pxr/imaging/plugin/hdRpr/volume.cpp

```diff
@@ -20,9 +23,9 @@ TF_DEFINE_PRIVATE_TOKENS(
 
 namespace
 {
-	const float defaultDencity = 1.f;
+	const float defaultDensity = 1.f;
 	const float defaultAlbedo = 1.f;
-	GfVec4f defaultColor = GfVec4f(35.0f, 35.0f, 35.f, 1.0f);
+	GfVec4f defaultColor = GfVec4f(10.0f, 0.0f, 0.0f, 1.0f);
```

Comment:
Is there a significance to this color?  Or is this just to test?  Also it's somewhat odd to have albedos > 1.0